<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>710</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>131</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="buttonLayout">
    <item>
     <widget class="QPushButton" name="historyDataButton">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>历史数据</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="parameterSettingButton">
      <property name="text">
       <string>参数调整</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="warningSettingButton">
      <property name="text">
       <string>预警管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="softwareInfoButton">
      <property name="text">
       <string>软件信息</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>500</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 245);</string>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="historyDataPage">
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(225, 225, 225, 255), stop:1 rgba(100, 100, 100, 255));</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="historyDataGroupBox">
       <property name="title">
        <string>历史数据</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QListWidget" name="historyDataListWidget"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="parameterSettingPage">
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(225, 225, 225, 255), stop:1 rgba(100, 100, 100, 255));</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="parameterSettingGroupBox">
       <property name="title">
        <string>参数详情</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="gaussSizeLabel">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>高斯模糊和大小</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="thresholdLabel">
              <property name="text">
               <string>二分图阈值</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="closingSizeLabel">
              <property name="text">
               <string>闭操作大小</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="maxAeraLabel">
              <property name="text">
               <string>目标物面积最大值</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="minAeraLabel">
              <property name="text">
               <string>目标物面积最小值</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="rateLabel">
              <property name="text">
               <string>尺度转换因子CF</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTextEdit" name="gaussSizeTextEdit"/>
            </item>
            <item>
             <widget class="QTextEdit" name="thresholdTextEdit"/>
            </item>
            <item>
             <widget class="QTextEdit" name="closingSizeTextEdit"/>
            </item>
            <item>
             <widget class="QTextEdit" name="maxAeraTextEdit"/>
            </item>
            <item>
             <widget class="QTextEdit" name="minAeraTextEdit"/>
            </item>
            <item>
             <widget class="QTextEdit" name="rateTextEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QFrame" name="frame">
            <property name="minimumSize">
             <size>
              <width>452</width>
              <height>220</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>452</width>
              <height>200</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="paraConfirmPushButton">
             <property name="geometry">
              <rect>
               <x>140</x>
               <y>10</y>
               <width>171</width>
               <height>51</height>
              </rect>
             </property>
             <property name="text">
              <string>确认修改</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="warningSettingPage">
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(225, 225, 225, 255), stop:1 rgba(100, 100, 100, 255));</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QGroupBox" name="warningSettingGroupBox">
       <property name="title">
        <string>预警设置</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QGroupBox" name="currentUserInfoGroupBox">
            <property name="title">
             <string>当前用户</string>
            </property>
            <widget class="QTextBrowser" name="currentUserInfoTextBrowser">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>431</width>
               <height>91</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QGroupBox" name="userManagementGroupBox">
            <property name="title">
             <string>添加用户</string>
            </property>
            <widget class="QTextEdit" name="addUserInfoTextEdit">
             <property name="geometry">
              <rect>
               <x>140</x>
               <y>20</y>
               <width>301</width>
               <height>41</height>
              </rect>
             </property>
            </widget>
            <widget class="QTextEdit" name="deleteUserInfoTextEdit">
             <property name="geometry">
              <rect>
               <x>140</x>
               <y>70</y>
               <width>301</width>
               <height>41</height>
              </rect>
             </property>
            </widget>
            <widget class="QPushButton" name="addUserButton">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>121</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>添加</string>
             </property>
            </widget>
            <widget class="QPushButton" name="deleteUserInfoButton">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>70</y>
               <width>121</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QGroupBox" name="warningMessageSettingGroupBox">
            <property name="title">
             <string>预警信息设置</string>
            </property>
            <widget class="QTextEdit" name="warningMessageSettingTextEdit">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>431</width>
               <height>101</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QGroupBox" name="settingConfirmGroupBox">
            <property name="title">
             <string/>
            </property>
            <widget class="QWidget" name="verticalLayoutWidget_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>251</width>
               <height>91</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QCheckBox" name="sendWarningMessageCheckBox">
                <property name="text">
                 <string>开启短信预警</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="interfaceWarningOnCheckBox">
                <property name="text">
                 <string>开启界面预警</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QPushButton" name="settingConfirmButton">
             <property name="geometry">
              <rect>
               <x>280</x>
               <y>30</y>
               <width>161</width>
               <height>51</height>
              </rect>
             </property>
             <property name="text">
              <string>确认</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="softwareInfoPage">
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(225, 225, 225, 255), stop:1 rgba(100, 100, 100, 255));</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QGroupBox" name="softwareInfoGroupBox">
       <property name="title">
        <string>软件相关信息</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QTextBrowser" name="softwareInfoTextBrowser">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:16pt;&quot;&gt;开发团队：邵子千 王洪宇 聂永璋&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:16pt;&quot;&gt;指导教师：徐文涛&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:16pt;&quot;&gt;软件信息：开发完成于2021.09.27&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
